content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-15.md

```diff
@@ -0,0 +1,394 @@
+    {{% /tab %}}
+    {{< /tabs >}}
+
+## Upgrade the first control plane node
+
+1.  On your first control plane node, upgrade kubeadm:
+
+    {{< tabs name="k8s_install_kubeadm_first_cp" >}}
+    {{% tab name="Ubuntu, Debian or HypriotOS" %}}
+    # replace x in 1.15.x-00 with the latest patch version
+    apt-mark unhold kubeadm && \
+    apt-get update && apt-get install -y kubeadm=1.15.x-00 && \
+    apt-mark hold kubeadm
+    {{% /tab %}}
+    {{% tab name="CentOS, RHEL or Fedora" %}}
+    # replace x in 1.15.x-0 with the latest patch version
+    yum install -y kubeadm-1.15.x-0 --disableexcludes=kubernetes
+    {{% /tab %}}
+    {{< /tabs >}}
+
+1.  Verify that the download works and has the expected version:
+
+    ```shell
+    kubeadm version
+    ```
+
+1.  On the control plane node, run:
+
+    ```shell
+    sudo kubeadm upgrade plan
+    ```
+
+    You should see output similar to this:
+
+    ```shell
+    [upgrade/config] Making sure the configuration is correct:
+    [upgrade/config] Reading configuration from the cluster...
+    [upgrade/config] FYI: You can look at this config file with 'kubectl -n kube-system get cm kubeadm-config -oyaml'
+    [preflight] Running pre-flight checks.
+    [upgrade] Making sure the cluster is healthy:
+    [upgrade] Fetching available versions to upgrade to
+    [upgrade/versions] Cluster version: v1.14.2
+    [upgrade/versions] kubeadm version: v1.15.0
+
+    Components that must be upgraded manually after you have upgraded the control plane with 'kubeadm upgrade apply':
+    COMPONENT   CURRENT       AVAILABLE
+    Kubelet     1 x v1.14.2   v1.15.0
+
+    Upgrade to the latest version in the v1.15 series:
+
+    COMPONENT            CURRENT   AVAILABLE
+    API Server           v1.14.2   v1.15.0
+    Controller Manager   v1.14.2   v1.15.0
+    Scheduler            v1.14.2   v1.15.0
+    Kube Proxy           v1.14.2   v1.15.0
+    CoreDNS              1.3.1     1.3.1
+    Etcd                 3.3.10    3.3.10
+
+    You can now apply the upgrade by executing the following command:
+
+        kubeadm upgrade apply v1.15.0
+
+    _____________________________________________________________________
+    ```
+
+    This command checks that your cluster can be upgraded, and fetches the versions you can upgrade to.
+
+{{< note >}}
+With the release of Kubernetes v1.15, `kubeadm upgrade` also automatically renews
+the certificates that it manages on this node. To opt-out of certificate renewal the flag `--certificate-renewal=false` can be used. For more information see the
+[certificate management guide](/docs/tasks/administer-cluster/kubeadm/kubeadm-certs).
+{{</ note >}}
+
+1.  Choose a version to upgrade to, and run the appropriate command. For example:
+
+    ```shell
+    sudo kubeadm upgrade apply v1.15.x
+    ```
+
+    - Replace `x` with the patch version you picked for this upgrade.
+
+    You should see output similar to this:
+
+    ```shell
+    [preflight] Running pre-flight checks.
```

Comment:
I'm not sure this is the latest output, I don't see certificate renewal messages...